packages/test-utils/src/test-utils/seedWallet.ts

```diff
@@ -13,13 +13,14 @@ export async function seedWallet(
   options: SeedWalletOptions = {},
 ) {
   const fuelProvider = await Provider.create(fuelProviderUrl);
-  const { minGasPrice } = await fuelProvider.getGasConfig();
+  const gasPrice = await fuelProvider.getLatestGasPrice();
+  const baseAssetId = fuelProvider.getBaseAssetId();
```

Comment:
@LuizAsFight Is this the expected approach or should we be accepting base asset id as a parameter?

Reply:
about the `baseAssetId` its correct, but about the `gasPrice` I think its not needed as a param anymore for the transfer function, can you check if we can just remove the gasPrice thing?